mill-mima/src/com/github/lolgab/mill/mima/Mima.scala

```diff
@@ -10,45 +10,64 @@ import mill.scalalib._
 import mill.scalalib.api.Util.scalaBinaryVersion
 
 trait Mima extends ScalaModule {
+
   def mimaPreviousArtifacts: Target[Agg[Dep]]
 
   def mimaCheckDirection: Target[CheckDirection] = T { CheckDirection.Backward }
 
+  def resolvedMimaPreviousArtifacts: T[Agg[PathRef]] = T {
+    resolveDeps(T.task {
+      mimaPreviousArtifacts().map(_.exclude("*" -> "*"))
+    })()
+  }
+
   def mimaReportBinaryIssues(): Command[Unit] = T.command {
     sanityCheckScalaVersion(scalaVersion())
     val log = T.ctx().log
-    val mimaLib = new MiMaLib(runClasspath().map(_.path.toIO))
-    val resolvedMimaPreviousArtifacts =
-      resolveDeps(T.task {
-        mimaPreviousArtifacts().map(_.exclude("*" -> "*"))
-      })()
-    val classes = compile().classes.path.toIO
+    val mimaLib =
+      new MiMaLib(runClasspath().map(_.path).filter(os.exists).map(_.toIO))
+    val classes = compile().classes.path // .toIO
```

Comment:
There is a leftover comment.
```suggestion
    val classes = compile().classes.path
```